README_EN.md

```diff
@@ -0,0 +1,45 @@
+# Tebukuro
+[![wercker status](https://app.wercker.com/status/35e08da3b0a742e5c50ffac2670a821b/s/master "wercker status")](https://app.wercker.com/project/byKey/35e08da3b0a742e5c50ffac2670a821b)
+
+A 'Mobile-friendly' open-source web service for events and communities management
+
+<img src="http://4.bp.blogspot.com/-dMHKByNfZxo/Uku9_iuHWHI/AAAAAAAAYqg/LjlcTLrN44o/s800/winter_tebukuro.png" width="320px">
+
+## What is tebukuro?
+
+Tebukuro is the 'Mobile-friendly' open-source web service, enabling you to manage everything about your communities and events only from mobile devices. Furthermore, supporting communication while holding events and after closed events, not only annoucements and receptions before events.
+
+## Features
+
+- **Mobile-friendly**
+  - [ ] Managing communities and events from mobile devices.
+
+- **Energizing your communities**
+  - [ ] Simple procedure for releasing new events to communities' site.
```

Comment:
I meant that our users can easily add application forms for events in their community's page, inside Tebukuro. @murajun1978, is that correct understanding? 

> Tebukuroのコンセプト
>- モバイルフレンドリー
>- モバイルのみで簡単にコミュニティやイベントを公開できる
>- イベント中の写真をアップロード（SNS通知）
>- コミュニティサイトに申し込みページを簡単に追加できる
>- プライベートなイベントを公開できる